src/pybind/CMakeLists.txt

```diff
@@ -43,39 +43,46 @@ else()
   target_link_libraries(pywrapper PRIVATE pybind11::embed)
 endif()
 
-# Note that LTO causes link time errors with GCC 8. To avoid this, we disable LTO for pybind using
-# NO_EXTRAS. See #266.
-pybind11_add_module(
-  _nmodl
-  NO_EXTRAS
-  ${NMODL_PROJECT_SOURCE_DIR}/src/ast/ast_common.hpp
-  ${NMODL_PROJECT_SOURCE_DIR}/src/pybind/pybind_utils.hpp
-  ${NMODL_PROJECT_SOURCE_DIR}/src/pybind/pynmodl.cpp
-  ${PYBIND_GENERATED_SOURCES}
-  $<TARGET_OBJECTS:symtab_obj>
-  $<TARGET_OBJECTS:visitor_obj>
-  $<TARGET_OBJECTS:lexer_obj>
-  $<TARGET_OBJECTS:util_obj>
-  $<TARGET_OBJECTS:printer_obj>)
+# avoid _nmodl target if python bindings are disabled
```

Comment:
Can't you avoid including `src/pybind/CMakeLists.txt` if `NMODL_ENABLE_PYTHON_BINDINGS` is disabled instead?

Reply:
@tristan0x : not really because we still use pybind11 `pybind11::initialize_interprete` interface to execute python code for Sympy solver.